docs/conf.py

```diff
@@ -545,6 +545,6 @@
 
 
 def setup(app: Sphinx) -> None:
-    app.add_config_value("revision", "", True)
+    app.add_config_value("revision", "", rebuild=True)
```

Comment:
Fixing a "boolean trap", where a boolean is used as an argument without the corresponding parameter name.